packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -1437,7 +1448,7 @@ contract Deploy is Deployer {
             string[] memory commands = new string[](3);
             commands[0] = "bash";
             commands[1] = "-c";
-            commands[2] = string.concat("[[ -f ", filePath, " ]] && echo \"present\"");
+            commands[2] = string.concat("[[ -f ", filePath, ' ]] && echo "present"');
```

Comment:
This looks like an unnecessary change. Did you do it intentionally or did an autoformatter do this?